packages/core/src/utils/indyIdentifiers.ts

```diff
@@ -0,0 +1,24 @@
+// For the definitions below see also: https://hyperledger.github.io/indy-did-method/#indy-did-method-identifiers
+export type Did = 'did'
+export type DidIndyMethod = 'indy'
+// Maybe this can be typed more strictly than string. Choosing string for now as this can be eg just `sovrin` or eg `sovrin:staging`
+export type DidIndyNamespace = string
+//  NOTE: because of the ambiguous nature - whether there is a colon or not within DidIndyNamespace this is the substring after the ***last*** colon
+export type NamespaceIdentifier = string
+
+// TODO: This template literal type can possibly be improved. This version leaves the substrings as potentially undefined
+export type IndyNamespace = `${Did}:${DidIndyMethod}:${DidIndyNamespace}:${NamespaceIdentifier}`
+
+export function createUnqualifiedIdentifier(qualifiedIdentifier: string): string {
```

Comment:
or maybe `unqualifyIndyDid` if it is getting to long. but identifier is maybe a bit misleading, we're working with dids here

```suggestion
export function getUnqualifiedDidFromQualifiedIndyDid(qualifiedIndyDid: string): string {
```

Reply:
I'm for unqualifyIndyDid